shaders/lang/ru_RU.lang

```diff
@@ -61,684 +120,1432 @@ screen.Direct_Light = Прямой свет
+            value.entityShadowDistanceMul.0.50 = 50% от Дистанции Теней
+            value.entityShadowDistanceMul.0.75 = 75% от Дистанции Теней
+            value.entityShadowDistanceMul.1.00 = 100% от Дистанции Теней
+
+|/////Shadow Filtering Settings/////|
+
+    screen.Filtering = Настройка Фильтра Теней
+
+[-----Basic Shadow Filter Samples-----]
+
+        option.SHADOW_FILTER_SAMPLE_COUNT = Обычный Образец Фильтра Теней
+
+[-----Minimum Shadow filter Radius-----]
+
+        option.Min_Shadow_Filter_Radius = Минимальный Радиус Фильтра Теней
+
+[-----Variable Penumbra Shadows-----]
+
+        option.Variable_Penumbra_Shadows = Переменные Полутеневые Тени
+
+[-----VPS Filter Samples-----]
+
+        option.VPS_Search_Samples = Образцы Фильтра ППТ (VPS)
+
+[-----Maximum Shadow Filter Radius-----]
+
+        option.Max_Shadow_Filter_Radius = Максимальный Радиус Фильтра Теней
+
+[-----Sun Size Factor-----]
+
+        option.Max_Filter_Depth = Фактор Размера Солнца
+
+|/////FloodFill/////|
+
+    screen.LPV = НаливноеЗаполнение (FloodFill) - §cОБЯЗАТЕЛЬНО IRIS§r
+
+[-----Enabled-----]
+
+        option.LPV_ENABLED = Включён
+
+[-----Size-----]
+
+        option.LPV_SIZE = Размер
+            value.LPV_SIZE.6 = Маленький [64]
+            value.LPV_SIZE.7 = Средний [128]
+            value.LPV_SIZE.8 = Большой [256]
+
+[-----Light Saturation-----]
+
+        option.LPV_SATURATION = Насыщенность Света
+            suffix.LPV_SATURATION=%
+
+[-----Tint Saturation-----]
+
+        option.LPV_TINT_SATURATION = Насыщенность Тонировки (Tint)
+            suffix.LPV_TINT_SATURATION=%
+
+[-----Normal Strength-----]
+
+        option.LPV_NORMAL_STRENGTH = Сила Нормалей
+            suffix.LPV_NORMAL_STRENGTH = %
+            value.LPV_NORMAL_STRENGTH.0 = ВЫКЛ
+
+[-----Entity Lights-----]
+
+        option.LPV_ENTITY_LIGHTS = Свет Сущностей
+
+[-----Redstone Lights-----]
+
+        option.LPV_REDSTONE_LIGHTS = Свет Редстоуна
+
+[-----Colored Candles-----]
+
+        option.LPV_COLORED_CANDLES = Цветные Свечи
+
+[-----Iris <= 1.7.0 Fix-----]
+
+        option.LPV_NOSHADOW_HACK = Исправление Iris <= 1.7.0
+
+[-----Fog Light Propagation-----]
+
+        option.LPV_VL_FOG_ILLUMINATION = Распространение Света Тумана
+
+[-----Fog Light Propagation Brightness-----]
+
+        option.LPV_VL_FOG_ILLUMINATION_BRIGHTNESS = Яркость Распространение Света Тумана
+            suffix.LPV_VL_FOG_ILLUMINATION_BRIGHTNESS=%
+
+|/////Sub-Surface Scattering/////|
+
+screen.Subsurface_Scattering = Под-Поверхностное Рассеяние
+
+[-----LabSSS Curve-----]
+
+    option.LabSSS_Curve = Кривость LabППР (SSS)
+
+[-----SSS for random blocks-----]
+
+    option.MISC_BLOCK_SSS = ППР (SSS) для разных блоков
+
+[-----SSS for mobs-----]
+
+    option.MOB_SSS = ППР (SSS) для мобов
+
+[-----SSS from the sky-----]
+
+    option.Ambient_SSS = ППР (SSS) от неба
+
+[-----Sky SSS Brightness-----]
+
+    option.ambientsss_brightness = Яркость Неба ППР (SSS)
```

Comment:
```suggestion
    option.ambientsss_brightness = Яркость SSS Неба
```

Reply:
Noun + Adjective is better than Adjective + Noun unless it's English (I won't remove the parenthesis).

I thought for a long time here because I thought your correction was good but then I remembered the rules of the Russian language and it didn't appeal to me.